library/src/main/java/com/google/android/exoplayer2/text/Cue.java

```diff
@@ -134,10 +143,26 @@
   /**
    * The size of the cue box in the writing direction specified as a fraction of the viewport size
    * in that direction, or {@link #DIMEN_UNSET}.
+   * <p>
+   * If {@link #bitmap} is not null then this value is used to indicate the width
    */
   public final float size;
 
   /**
+   * The height size of the cue box when a {@link #bitmap} is set specified as a fraction of the
+   * viewport size in that direction, or {@link #DIMEN_UNSET}.
+   */
+  public final float size_height;
```

Comment:
Trying to get this straight in my head again.

We can grab width and height from the bitmap, but the passed in width (size) and height (size_height) should be the fractional value of the width and height based on the original plane width and height.(correct?).  So a image sub from a 1080p plane would pass in the percentage of the screen the bitmap takes up so it can be resized on a 720p painter view...  Hard to test because everything I have is 1080p.

Would it be better to change the parameter to plane_height (and add one for plane_width and not use size) and do all the calculations in subtitlepainter?